sdk/identity/azure-identity/azure/identity/_internal/msal_credentials.py

```diff
@@ -29,7 +29,7 @@ def __init__(
             **kwargs
     ) -> None:
         authority = kwargs.pop("authority", None)
-        # self._validate_authority = kwargs.pop("validate_authority", True)
```

Comment:
Is this something we plan to revert at some point? Why comment this out instead of removing it?

Reply:
I think I deleted this line?

Reply:
You're completely right, and I'm dumb 🙂